ext/spl/spl_observer.c

```diff
@@ -1304,6 +1314,7 @@ static const zend_function_entry spl_funcs_MultipleIterator[] = {
 	SPL_MA(MultipleIterator,  detachIterator,         SplObjectStorage, detach,   arginfo_MultipleIterator_detachIterator,   0)
 	SPL_MA(MultipleIterator,  containsIterator,       SplObjectStorage, contains, arginfo_MultipleIterator_containsIterator, 0)
 	SPL_MA(MultipleIterator,  countIterators,         SplObjectStorage, count,    arginfo_splobject_void,                    0)
+	SPL_ME(SplObjectStorage,  __debugInfo,            arginfo_splobject_void,                     0)
```

Comment:
Should use SPL_MA here?

Reply:
Yes. I've left some other occurrences (e.g. SplPriorityQueue) as SPL_ME(), because they already use SPL_ME() instead of SPL_MA().